src/main/resources/default/assets/common/core.js

```diff
@@ -1,11 +1,11 @@
-// Checks if the given string is empty or null.
-sirius.isEmpty = function (aString) {
-    return aString === null || aString === undefined || aString === '';
+// Checks if the given value is empty or null.
+sirius.isEmpty = function (value) {
+    return value === null || value === undefined || typeof value === 'undefined' || value === '';
```

Comment:
Should that trim value before the check?

Reply:
I don't know. Opinions?

Reply:
I prefer to be consistent with [`Strings.isEmpty`](https://github.com/scireum/sirius-kernel/blob/48bc76c7459b9bbeaeb9cc15eedf986f524f5fd0/src/main/java/sirius/kernel/commons/Strings.java#L115) which doesn't do a trim